src/GAP.jl

```diff
@@ -53,6 +53,18 @@ const GAP_VERSION = VersionNumber(sysinfo["GAP_VERSION"])
 
 include("types.jl")
 
+"""
+    GAP.@include(path)
+
+Read and execute the GAP code in the file at `path`,
+which is interpreted as a relative path relative to the file where this macro is used.
+This is similar to julia's built-in `include` function,
+but for GAP code instead of Julia code.
```

Comment:
I was concerned whether `GAP.@include("/absolute/path/file.G")` works (if not we should warn users). But I think it will:
```
julia> normpath(@__DIR__, "file.jl")
"/Users/mhorn/Projekte/OSCAR/GAP.jl/file.jl"

julia> normpath(@__DIR__, "/tmp/file.jl")
"/tmp/file.jl"
```
Very nice!